packages/core/src/rum/instrumentation/resourceTracking/distributedTracing/__tests__/distributedTracing.test.ts

```diff
@@ -9,63 +12,98 @@ describe('TraceIdentifier', () => {
 
         // WHEN
         while (counter-- > 0) {
-            generatedIds.add(new TraceIdentifier().toString(10));
+            generatedIds.add(
+                new TracingIdentifier().toString(
+                    TracingIdRepresentation.decimal
+                )
+            );
         }
 
         // THEN
         expect(generatedIds.size).toBe(iterations);
     });
 
-    it('M return an 64 bits positive integer W toString', async () => {
+    it('M return a valid 128 bits positive integer W toString', async () => {
         let iterations = 100;
         while (iterations-- > 0) {
             // GIVEN
-            const id = new TraceIdentifier().toString(10);
+            const id = new TracingIdentifier();
+            const idStr128 = id.toString(TracingIdRepresentation.decimal);
 
             // THEN
-            expect(id).toMatch(/[1-9]{1,19}/);
-            // should be less than the max 64 bits integer
-            if (id.length === 19) {
-                expect(id < '9223372036854775807').toBeTruthy();
-            }
+            expect(idStr128).toMatch(/[1-9]{1,19}/);
+            expect(TracingIdentifier.is64Bits(idStr128)).toBe(false);
+            expect(TracingIdentifier.is128Bits(idStr128)).toBe(true);
         }
     });
 
-    it('M return an 64 bits positive hex W toString(16)', async () => {
+    it('M return a valid 64 bits low and high part integer W toString', async () => {
         let iterations = 100;
         while (iterations-- > 0) {
             // GIVEN
-            const trace = new TraceIdentifier();
-            const id = trace.toString(16);
-            const paddedId = trace.toPaddedString(16, 16);
+            const id = new TracingIdentifier();
+            const idStrLow64 = id.toString(TracingIdRepresentation.lowDecimal);
+            const idStrHigh64 = id.highToString(10);
```

Comment:
Any reason to have `highToString` as a separate method? Couldn't you have `TracingIdRepresentation.highDecimal`?